### ISSUE TITLE
Slightly broken Tooltips and Menu Characters

### ISSUE THREAD

**Issue by SalmonSays** [InnovativeOnlineIndustries/Emojiful#18]
I've discovered a strange mod interaction on 1.16.5 Forge 36.0.01 between "emojiful-1.16.4-2.1.2.jar" and "ModernUI-1.16.5-2.4.1.76-universal.jar" of ModernUI (https://www.curseforge.com/minecraft/mc-mods/modern-ui)

This manifests as some sort of invisible character prefixes turning into garbage text with a "§" character code on tooltips and even the main menu, pictured here:

![wzetwFk](https://user-images.githubusercontent.com/11697389/105471669-9ddc2c00-5c60-11eb-878e-a24e6a4eba7c.png)
![gTJaPIY](https://user-images.githubusercontent.com/11697389/105471670-9ddc2c00-5c60-11eb-8eec-2137461bf7bf.png)

**Comment by Buuz135** [comment]
I dont think that emojiful works with modern ui as it replaces the font renderer too. Does it happen without emojiful too?

**Comment by SalmonSays** [comment]
Nope, it's definitely an interaction between the two. Is there any way to prevent them from interfering with each other's additions/modifications?

**Comment by Buuz135** [comment]
Do emojis still work?

**Comment by SalmonSays** [comment]
Emojis do still work/take priority, none of the obvious text/font modification of ModernUI does though. As an addendum- the Menu's Forge Beta Warning is an indicator that the interaction between the two is wonky, but seeing "wonky" tooltips requires further installation of JEI or equivalent.

**Comment by SalmonSays** [comment]
The ModernUI mod dev chimed in over on Discord a few hours ago-

"Modern UI will not be compatible with it" (In response to the initial issue report and narrowing down conflicting mods)
"But we could provide emoji features"
"I think you should advise them to use their font renderer only in the chat screen, I will provide an API for embedding images in text, and web pictures"

I'm cautiously hopeful we can get the best of both worlds, in that the combined features of your mods are really no-brainers and feel like they should be baked into vanilla MC.

**Comment by DatsiK96** [comment]
Any update on this?

**Comment by EnterFor** [comment]
Any update on this?

**Comment by Ashley123477** [comment]
Still broken as of now

**Comment by X-Niter** [comment]
As of now +1, Still a thing. Only for confirmation
